frontend/src/Style/Cursor.elm

```diff
@@ -0,0 +1,15 @@
+module Style.Cursor exposing (notAllowed)
+
+
+type CursorTypes
```

Comment:
This type is unnecessary, just define return the class name in the below functions.